### PR TITLE
feat: add Proposal settings (excl. validation)

### PR DESCRIPTION
**Pull request by Sekhmet**
### Summary

This PR adds proposal settings (excluding validation as this will be similar to proposal validation so it will be done in single PR).

### How to test

1. Go to your offchain space settings.
2. You can modify proposal settings.

### Screenshot

image

### CODE REVIEW

**Review by wa0x6e — CHANGES_REQUESTED** [review]
After dirtying the "guidelines" input with something invalid, the input stay in the error state even after clearing its content

**Review by wa0x6e — CHANGES_REQUESTED** [review]
When errored, this tab does not show the bottom toolbar with the reset button

**Review by ChaituVR — COMMENTED** [review]
tAck

**Comment by Sekhmet** [comment]
Most things didn't trigger bottom bar with reset, but I think it makes sense to do, now made it so every page with internal validation will trigger bottom bar with error with reset.

File: apps/ui/src/views/Space/Settings.vue [hunk 1]
```diff
@@ -449,6 +471,7 @@ watchEffect(() => setTitle(`Edit settings - ${props.space.name}`));
         (isModified &&
           canModifySettings &&
           !hasAdvancedErrors &&
+          !hasProposalErrors &&
```
[the remark this answers is not in the document]
Comment (Sekhmet) [reply apps/ui/src/views/Space/Settings.vue]: Added in this PR.

**Review by wa0x6e — APPROVED** [review]
tAck,
but we still have the global issue about form error state not persisting after a tab change. Not sure that adding a fix on each form is the way, may be better to ignore them for now and wait for the dirty composable

**Pull request merged**